pkg/syncer/sync.go

```diff
@@ -112,33 +113,29 @@ func (s *Syncer) SyncPendingCharts(names ...string) error {
 // updating the images in values.yaml. The local chart must include an image hints file so relok8s library knows how to
 // update the images
 func (s *Syncer) SyncWithRelok8s(chart *Chart, outdir string) (string, error) {
-	// Once https://github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/issues/94 is solved, we could
-	// specify the name we want for the output file. Until then, we should keep using this template thing
-	outputChartPath := filepath.Join(outdir, "%s-%s.relocated.tgz")
-	packagedChartPath := filepath.Join(outdir, fmt.Sprintf("%s-%s.relocated.tgz", chart.Name, chart.Version))
-	req := &mover.ChartMoveRequest{
-		Source: mover.Source{
-			Chart: mover.ChartSpec{
-				Local: &mover.LocalChart{
-					// This chart has a .relok8s-images.yaml file inside so no need to explicitly pass that
-					Path: chart.TgzPath,
-				},
-			},
-		},
-		Target: mover.Target{
-			Rules: mover.RewriteRules{
-				Registry:         s.target.GetContainerRegistry(),
-				RepositoryPrefix: s.target.GetContainerRepository(),
-				ForcePush:        true,
-			},
-			Chart: mover.ChartSpec{
-				Local: &mover.LocalChart{
-					// output will be [chart-name]-[chart-version].relocated.tgz
-					Path: outputChartPath,
-				},
-			},
-		},
+	packagedChartPath := ""
+	req := &mover.ChartMoveRequest{}
+	if s.target.GetIntermediateBundlesPath() != "" {
```

Comment:
it might make sense to extract the logic that calculates the relok8s request to its own method so it's simpler to test.

```
func getRelok8sMoveRequest(source *Syncer.Source, target *Syncer.Target, chart *chart.Chart) *mover.ChartMoveRequest 
```